odl/tomo/analytic/filtered_back_projection.py

```diff
@@ -324,10 +331,13 @@ def fbp_filter_op(ray_trafo, padding=True, filter_type='Ram-Lak',
         If the data space should be zero padded. Without padding, the data may
         be corrupted due to the circular convolution used. Using padding makes
         the algorithm slower.
-    filter_type : string, optional
-        The type of filter to be used. The options are, approximate order from
-        most noise senstive to least noise sensitive: 'Ram-Lak', 'Shepp-Logan',
-        'Cosine', 'Hamming' and 'Hann'.
+    filter_type : optional
+        The type of filter to be used.
+        The predefined options are, in approximate order from most noise
+        senstive to least noise sensitive:
+        `'Ram-Lak'`, `'Shepp-Logan'`, `'Cosine'`, `'Hamming'` and `'Hann'`.
```

Comment:
Should be double backticks here. You can also make it one listing enclosed by double backticks.